test/tcp_tunnel.js

```diff
@@ -42,16 +43,16 @@ const closeServer = (server, connections) => new Promise((resolve, reject) => {
 
 describe('tcp_tunnel.createTunnel', () => {
     it('throws error if proxyUrl is not in correct format', () => {
-        assert.throws(() => { createTunnel('socks://user:password@whatever.com:123', 'localhost:9000'); }, /must have the "http" protocol/);
-        assert.throws(() => { createTunnel('socks5://user:password@whatever.com', 'localhost:9000'); }, /must have the "http" protocol/);
+        expectThrowsAsync(async () => { await createTunnel('socks://user:password@whatever.com:123', 'localhost:9000'); }, /must have the "http" protocol/);
```

Comment:
```suggestion
        await expect(createTunnel('socks://user:password@whatever.com:123', 'localhost:9000')).rejects.toThrow(/must have the "http" protocol/);
```

Would this work here? This is the syntax we're using e.g. in `apify-core`.

Reply:
This is sadly not `jest` but `chai` :(

Reply:
Ah, okay then :cry: